src/Symfony/Component/RateLimiter/Tests/Strategy/TokenBucketLimiterTest.php

```diff
@@ -9,16 +9,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\RateLimiter\Tests;
+namespace Symfony\Component\RateLimiter\Strategy\Tests;
```

Comment:
wrong change. Should be `Tests\Strategy`

Reply:
Thank you